src/dnx/dnx.cpp

```diff
@@ -5,395 +5,401 @@
 
 void GetModuleDirectory(HMODULE module, LPWSTR szPath)
 {
-    DWORD dirLength = GetModuleFileName(module, szPath, MAX_PATH);
-    for (dirLength--; dirLength >= 0 && szPath[dirLength] != '\\'; dirLength--);
-    szPath[dirLength + 1] = L'\0';
+	DWORD dirLength = GetModuleFileName(module, szPath, MAX_PATH);
+	for (dirLength--; dirLength >= 0 && szPath[dirLength] != '\\'; dirLength--);
+	szPath[dirLength + 1] = L'\0';
```

Comment:
The default VS C/C++ Tabs setting is to use hard-tabs. This all appeared when I Ctrl-K Ctrl-D'd on the cpp file. If we want to change from the default, I'm fine with that, but it seems better to use the default.